### PR TITLE
PT-159405994 get channel contract state

### PR DESCRIPTION
**Pull request by cytadela8**
Documentation for aeternity/aeternity#2232

### CODE REVIEW

**Comment by velzevur** [comment]
Since [new encoding types](https://github.com/aeternity/aeternity/pull/2232/files#diff-3daa3c4348ace982101dff7711070ee4R198) are introduced, please add them to [node/api/api_encoding.md](https://github.com/aeternity/protocol/blob/master/node/api/api_encoding.md)
Otherwise looks good

File: node/api/channel_ws_api.md [hunk 1]
```diff
@@ -559,3 +559,63 @@ Roles:
...
+
+##### Request
+```javascript
+{'action': 'get', 'payload': {'pubkey': 'ct_2KYa2C7zGuQo3tX34vEsZyREspGzfmstGTVQZRdGNe9ow98X3u'}, 'tag': 'contract'}
+```
+
+##### Response
+```javascript
+{ 'channel_id': 'ch_2mXHiArGB2ar98bWhPKcZoXagLTh7JHWSeVbDd3p1S7s1cqtQ2',
+  'id': -576460752303423479,
```
Comment (lucafavatella) [inline node/api/channel_ws_api.md]: Negative? Could you please clarify?

File: node/api/channel_ws_api.md [hunk 2]
```diff
@@ -559,3 +559,63 @@ Roles:
...
+
+#### Response
+ | Field name | Value |
+ | ---- | ---- |
+ | `contract` | object with contract details |
+ | `contract->id` | contract id (equals to the requested pubkey) |
+ | `contract->owner_id` | contract owner id |
+ | `contract->vm_version` | contract vm version (integer) |
+ | `contract->abi_version` | contract ABI version (integer) |
+ | `contract->log` | contract event log (binary string) |
```
Comment (lucafavatella) [inline node/api/channel_ws_api.md]: Same as https://github.com/aeternity/aeternity/pull/2232/commits/ea483707495691fc569b5d0e1403dcb6d7e5bad1#r264274061

File: node/api/channel_ws_api.md (hunk 1 above)
Comment (cytadela8) [reply node/api/channel_ws_api.md]: This is part of json_rpc

Comment (cytadela8) [reply node/api/channel_ws_api.md]: But this shouldn't be a rpc example.

Comment (cytadela8) [reply node/api/channel_ws_api.md]: I will regenerate the example

File: node/api/channel_ws_api.md [hunk 3]
```diff
@@ -559,3 +559,58 @@ Roles:
   'tag': 'offchain_state',
   'version': 1}
 ```
+
+### Latest contract state
+ * **action:** `get`
+ * **tag:** `contract`
+ * **payload:**
+  | Name | Type | Description | Required |
```
Comment (lucafavatella) [inline node/api/channel_ws_api.md]:
```suggestion

  | Name | Type | Description | Required |
```

**Pull request merged**